### ISSUE TITLE
Show notification when running on rosetta

### ISSUE THREAD

**Issue by mrkn** [microsoft/vscode#127598]
Issue Type: <b>Feature Request</b>

I encountered a problem that can be avoidable if VSCode's Electron binary were replaced with that of universal binary version when the auto upgrading of VSCode.

The root cause of the problem is my carelessness that I persuaded that VSCode was upgraded to the universal binary version automatically.  But, I guess many beginners cannot notice this situation.

I've already upgrade VSCode to the universal binary version manually.

VS Code version: Code 1.57.1 (Universal) (507ce72a4466fbb27b715c3722558bb15afa9f48, 2021-06-17T13:28:32.912Z)
OS version: Darwin arm64 20.5.0
Restricted Mode: No

**Comment by deepak1556** [comment]
Can you clarify what the issue you are facing ?

Are you suggesting to replace the x64 version of vscode with universal version during update ? If so, why ?

**Comment by mrkn** [comment]
The problem I encountered was that clang generated x86-64 binary when I executed clang in the terminal on VSCode even though I use M1 Mac.  The cause of this problem is that the process of VSCode was running on Rosetta 2.

It was mysterious for me because VSCode I used is up to date.  I investigated why the process is running on Rosetta 2, and I found that the reason why was that the Electron executable was not a universal binary.  So, I guessed the VSCode updater was not replacing the Electron binary with its universal binary version.

I got my M1 Mac on this February, then I downloaded and installed VSCode the same time.  I guess people with the similar situation to me can face the same problem.
